src/tests/Parser/Brewmaster/Fixtures/SimpleFight.js

```diff
@@ -0,0 +1,60 @@
+import SPELLS from 'common/SPELLS';
+
+export const FIGHT_END = 11000;
+export const TOTAL_STAGGERED = 240;
+
+const thisPlayer = 1;
+const enemy = 2;
+
+export function processEvents(events, module, fightEndTime) {
+  if (!fightEndTime) {
+    fightEndTime = FIGHT_END;
+  }
+  events.forEach(event => {
+    if (event.timestamp <= fightEndTime) {
+      module.triggerEvent('toPlayer_' + event.type, event);
+      module.triggerEvent('byPlayer_' + event.type, event);
+    }
+  });
+}
+
+export const events = [
+  { type: "damage", sourceid: enemy, targetid: thisPlayer, amount: 400, timestamp: 1, ability: { guid: 1 } },
```

Comment:
I have renamed that one as SimpleFight as the name implies, and will create another array in another file to test other scenarios.